sdk/servicebus/service-bus/test/utils/testUtils.ts

```diff
@@ -370,19 +358,18 @@ export async function getSenderReceiverClients(
       };
     }
     case TestClientType.TopicFilterTestDefaultSubscription: {
-      if (env[EnvVarKeys.CLEAN_NAMESPACE]) {
```

Comment:
@ramya-rao-a Removed use of CLEAN_NAMESPACE flag per comment https://github.com/Azure/azure-sdk-for-js/issues/5985#issuecomment-569818862

See commit https://github.com/Azure/azure-sdk-for-js/pull/6603/commits/4c6e78d3edeb33ec5fbe1b54d4a1882c25a48ec9